packages/insomnia-importers/src/importers/curl.js

```diff
@@ -174,16 +174,23 @@ function importArgs(args) {
 
   // Body (Text or Blob)
   const bodyAsGET = getPairValue(pairs, false, 'G', 'get');
-  const textBody = getPairValue(
-    pairs,
-    null,
+
+  let textBodyParams = [];
+  for (const paramName of [
     'd',
     'data',
     'data-raw',
     'data-urlencode',
     'data-binary',
     'data-ascii',
-  );
+  ]) {
+    if (pairs[paramName] && pairs[paramName].length) {
+      textBodyParams = textBodyParams.concat(pairs[paramName]);
+    }
```

Comment:
We can tidy this one up to reduce the number of operations against an array, make it a little more performant by using `array.push` instead of `array.concat`. We also recently enabled optional chaining!

So with that, I suggest something like the following:

```suggestion
	const pair = pairs[paramName];
    if (pair?.length) {
  	  textBodyParams.push(pair);
    }
```

Reply:
Kidding, I totally misread this! `pairs` is a nested array. 👍